cmd/skuba/node/join.go

```diff
@@ -51,8 +51,11 @@ func NewJoinCmd() *cobra.Command {
 			}
 
 			joinConfiguration.Role = deployments.MustGetRoleFromString(joinOptions.role)
-
-			if err := node.Join(joinConfiguration, target.GetDeployment(nodenames[0])); err != nil {
+			clientSet, err := kubernetes.GetAdminClientSet()
+			if err != nil {
+				klog.Errorf("Can not retrieve the ClientSet from kubernetes: %v", err)
+			}
```

Comment:
[1]

Reply:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md
I would rather to change remove.go to make it consistent.